ui/apps/platform/cypress/helpers/ocpConsole.ts

```diff
@@ -0,0 +1,6 @@
+import pf6 from '../selectors/pf6';
+
+export function selectProject(project: string) {
+    cy.get(`.co-namespace-bar ${pf6.menuToggle}`).click();
+    cy.get(`.co-namespace-bar ${pf6.menuItem}:contains("${project}")`).click();
```

Comment:
Question, not necessarily change request. What is source of project names? Thinking how likely that a change outside of scope of test might select unintended project by partial match, as happened in Network Graph with `central` versus `central-db` when the latter appeared.